src/policy/ephemeral_policy.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2024-present The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include<policy/ephemeral_policy.h>
+#include<policy/policy.h>
+
+bool CheckValidEphemeralTx(const CTransactionRef tx, CFeeRate dust_relay_fee, CAmount base_fee, CAmount mod_fee, TxValidationState& state)
```

Comment:
I think this should be `const CTransactionRef& tx` (and in the header file as well)

Reply:
done